### ISSUE TITLE
[Feature Request] @open-wc/scoped-elements, make the tagName optional

### ISSUE THREAD

**Issue by RoXuS** [open-wc/open-wc#2433]
90% of the time we don’t change the tagName of the component.

Maybe it should be a good idea to add a “mode” on scopedElements to avoid tagName?

Something like:

```
static get scopedElements() {
  return {
    DtTooltip,
    DtNotification,
    'dt-tooltip': DtTooltip...
  }
}
```

**Comment by RoCat** [comment]
Is it intended to have a tagName in pascal/camel case like in react ? 
In my vision, all tag names should be kebab cased. 
If so, maybe we just could convert pascalized tag name to kebab case.

Other solution is to use Class.name property, or static get is, but it seems unset when one uses @customElement

**Comment by RoXuS** [comment]
Yes but technically we don't have to use @customElements with scopedElementMixin so we will not have customElements too.

**Comment by RoXuS** [comment]
yeah it's a good solution for us but that it's not a generic thing...
